cioos_data_transform/cioos_data_transform/utils/compare_files.py

```diff
@@ -0,0 +1,72 @@
+import hashlib
+import os
+import json
+import glob
+
+
+def get_file_stats(flist):
+    info = {}
+    for f in flist:
+        info.update({f: [os.stat(f).st_size, int(os.stat(f).st_mtime / 60)]})
```

Comment:
any reason for update() here? Could do `info[f]=[os.stat(f).st_size, int(os.stat(f).st_mtime / 60)]`

Reply:
good point, update() is not needed here...